great_expectations/core/validation_config.py

```diff
@@ -23,10 +20,25 @@ class ValidationConfig(BaseModel):
 
     """
 
+    class Config:
+        arbitrary_types_allowed = True
+        json_encoders = {
+            ExpectationSuite: lambda v: expectationSuiteSchema.dump(v),
+        }
```

Comment:
Concurrently using Marshmallow and Pydantic results in something like this. We need custom encoders/decoders but this should work as intended